clientd3d/ownerdrw.c

```diff
@@ -341,16 +342,39 @@ void DrawOwnerListItem(const DRAWITEMSTRUCT *lpdis, Bool selected, Bool combo)
    SetBkMode(lpdis->hDC, OPAQUE);
    obj = (object_node*)lpdis->itemData;
 
-   hColorBg = GetBrush(GetItemListColor(lpdis->hwndItem, (selected? SEL_BGD : UNSEL_BGD)));
+   hColorBg = GetBrush(
+                  GetItemListColor(
+                     lpdis->hwndItem,
+                     (selected? SEL_BGD : UNSEL_BGD),
+                     text_color_value));
```

Comment:
It looks like text_color_value can only be ITEM_TEXT_COLOR_NORMAL at this point?

Reply:
Yes it is essentially = 0 at this point.  These two functions are fetching the background color.

For my change, I am only affecting the foreground text color.  The logic of `GetItemListColor` first checks to see if the text_color_value is anything other than 0 or `ITEM_TEXT_COLOR_NORMAL`.  If it is not set, then it checks to see what the background color is.  If if it is set, then it will return the color for the foreground text.

In these two functions (which existed previously) I am passing it the parameter 0 or `ITEM_TEXT_COLOR_NORMAL` so that it will skip the foreground text color section.

Should I try and refactor `GetItemListColor`?  The logic works but I definitely need to pass 0 for the third parameter in some form currently.

Reply:
Adjusted default parameter as requested in other comments.